src/components/nav-user/nav-user.tsx

```diff
@@ -0,0 +1,107 @@
+import { Component, Element, h, Host, Prop, Event, EventEmitter } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-user",
+  styleUrl: "nav-user.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavUser {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteNavUserElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the text to display, such as a username or full name. */
+  @Prop() fullName: string;
+
+  /** Describes the appearance of the avatar. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the unique id of the user. */
+  @Prop() userId: string;
+
+  /** Specifies the username of the user. */
+  @Prop() username: string;
+
+  /** When `true`, hides the `fullName` and `username` contents. */
+  @Prop({ reflect: true }) textDisabled = false;
+
+  /** Specifies the `src` to an image to display in the Avatar (remember to add a token if the user is private). */
```

Comment:
> remember to add a token if the user is private

Without additional context, I think this seems out of place.

Reply:
Thoughts on adding additional context @geospatialem  ?

Reply:
I still think the token is important to note here. Maybe some context clarity can help?

Can we update a bit to fit the same description used in Avatar, where it specifies as an image.

Perhaps something like:

```
Specifies the `src` to an image (remember to add a token if the user is private).
```

Otherwise, if referencing Avatar makes sense here in the doc, how about the following:

```
Specifies the `src` of the `calcite-avatar` image (remember to add a token if the user is private).
```